brave/brave-core
# Share Sheet Edit is broken for custom items

Resolves https://github.com/brave/brave-browser/issues/36506

Looks like this is broken from the beginning but latest changes showed the menu item disappear instead of moving to top.
The activity type has to be set in order to have share sheet item work properly.



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Launch Brave 1.63 or newer
- Go to any website
- Tap the Share button > Edit Actions...
- Add Copy Clean Link to the Favorites > Done > Observe Copy Clean Link is not shown in the Favorites
- Tap Edit Actions... > Observe Copy Clean Link is not shown anymore
- Add Report a Broken Site > Done > Observe Report a Broken Site is properly added
